src/main/java/uk/ac/ebi/embl/converter/cli/CommandConversion.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2025 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.embl.converter.cli;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import io.vavr.Function0;
+import java.io.*;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import uk.ac.ebi.embl.converter.Converter;
+import uk.ac.ebi.embl.converter.exception.*;
+import uk.ac.ebi.embl.converter.exception.CLIException;
+import uk.ac.ebi.embl.converter.fftogff3.FFToGff3Converter;
+import uk.ac.ebi.embl.converter.gff3toff.Gff3ToFFConverter;
+import uk.ac.ebi.embl.converter.validation.RuleSeverityState;
+
+// Using pandoc CLI interface conventions
+@CommandLine.Command(name = "conversion", description = "Performs format conversions to or from gff3")
+public class CommandConversion implements Runnable {
```

Comment:
Suggestion to rename the class to
`public class FileConversionCommand implements Runnable {`